docs/shared/core-features/cache-task-results.md

```diff
@@ -53,48 +51,125 @@ the same output. As an example, e2e test runs that hit the backend API cannot be
 the result of the test run.
 {% /callout %}
 
-Now, run the following command twice. The second time the operation will be instant:
+Now, if you run a `build` task twice, the second time the operation will be instant because it is restored from the cache.
 
-```shell
-nx build header
+{% terminal-video src="/documentation/shared/images/caching/cache-terminal-animation.mp4" alt="Video showing the terminal output of running a build command first without cache and then with cache. The 2nd run is almost instant, taking just 18ms" /%}
```

Comment:
🤔 good point. I'll think about it. It could also be the same size video, but the terminal split horizontally. I'll have a look and submit a follow-up PR